src/app/services/login-service/login-service.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+
+  constructor(private http: HttpClient) { }
+  login(username, password): Observable<any> {
+    const headers = new HttpHeaders({
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*'
+    });
+    const options = {
+      headers: headers,
+      observe: 'response' as 'body'
+    };
+
+    return this.http.post(`${environment.loginUrl}`,
```

Comment:
any url should be composed of {hostname} and the {endpoint}
in this case it should be `${environment.hostName}/login`

Reply:
Fixed, thank you.